config/environments/production.rb

```diff
@@ -53,7 +53,16 @@
 
   # Use a different logger for distributed setups.
   # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
-
+  config.action_mailer.delivery_method = :smtp
+  # SMTP settings for gmail
+  config.action_mailer.smtp_settings = {
+   address:              "smtp.gmail.com",
```

Comment:
Unnecessary spacing detected.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.